traefik/values.yaml

```diff
@@ -343,6 +384,25 @@ service:
     # - 172.16.0.0/16
   externalIPs: []
     # - 1.2.3.4
+  # Options for Traefik internal service
+  internal:
+    # Enables internal Service creation, exposing ports that have their
+    # `.exposeInternal` set to `true`
+    enabled: false
+    # Service type for internal Service, defaults to `.Values.service.type`
+    # type: ClusterIP
+    # Additional labels for internal Service, defaults to `.Values.service.labels`
+    # labels: {}
+    # Additional annotations for internal Service, defaults to
+    # `.Values.service.annotations`
+    # annotations:
+    #   networking.gke.io/internal-load-balancer-subnet: gke-services
+    #   networking.gke.io/load-balancer-type: Internal
```

Comment:
May we please add MetalLB annotations as another example to use together with private IP pools? What do you think?
`metallb.universe.tf/address-pool: private-ips`

Reply:
Sure